src/vsr/superblock.zig

```diff
@@ -304,22 +300,30 @@ pub const SuperBlockHeader = extern struct {
         manifest_oldest_checksum: u128,
         manifest_newest_checksum: u128,
         snapshots_block_checksum: u128,
+        free_set_head_checksum: u128,
 
         manifest_oldest_address: u64,
         manifest_newest_address: u64,
         snapshots_block_address: u64,
+        free_set_head_address: u64,
 
         /// The last operation committed to the state machine. At startup, replay the log hereafter.
         commit_min: u64,
 
+        // Storage size in bytes.
+        storage_size: u64,
```

Comment:
Storage size is now part of checkpoint state, as it is no longer a function of the rest of the superblock, and we want to know storage size even without having access to the grid.